sw/device/sca/aes_serial/aes_serial.c

```diff
@@ -263,7 +259,7 @@ static void simple_serial_handle_command(const aes_cfg_t *aes_cfg,
   }
 
   if (cmd[0] == 'v') {
-    print_cmd_response('z', (const uint8_t *)&kSimpleSerialProtocolVersion,
+    print_cmd_response('z', (uint8_t[1]){kSimpleSerialProtocolVersion},
```

Comment:
This syntax looks weird to me, but I am not a C expert ;-)

Reply:
This is a [compound literal](https://en.cppreference.com/w/c/language/compound_literal). You may also want to check "6.5.2.5 Compound literals" in the C11 standard (p: 85-87). BTW, this is is a transient change, added here just to make it work with `enum` constants.